scipy-notebook/Dockerfile

```diff
@@ -64,7 +64,7 @@ RUN mamba install --yes \
 # Install facets which does not have a pip or conda package at the moment
 WORKDIR /tmp
 RUN git clone https://github.com/PAIR-code/facets.git && \
-    jupyter nbextension install facets/facets-dist/ --sys-prefix && \
+    jupyter nbclassic-extension install facets/facets-dist/ --sys-prefix && \
```

Comment:
It seems that the command name has changed.
https://discourse.jupyter.org/t/notebookv7-nbclassic-how-do-you-enable-disable-classic-extensions-from-the-command-line/16344

Reply:
For reference there is also a note in the docs: https://nbclassic.readthedocs.io/en/latest/nbclassic_dev_faq.html#noteworthy-updates-in-nbclassic